java/client/src/main/java/glide/api/commands/SortedSetBaseCommands.java

```diff
@@ -853,8 +859,8 @@ CompletableFuture<Long> zrangestore(
      * stores the result in <code>destination</code>. If <code>destination</code> already exists, it
      * is overwritten. Otherwise, a new sorted set will be created.
      *
-     * @apiNote When in cluster mode, <code>destination</code> and all <code>keys</code> must map to
-     *     the same <code>hash slot</code>.
+     * @apiNote When in cluster mode, <code>destination</code> and all <code>keysOrWeightedKeys</code>
```

Comment:
even if you dont agree with the previous places of this comment, here it's a must   
```suggestion
     * @apiNote When in cluster mode, <code>destination</code> and all keys in <code>keysOrWeightedKeys</code>
```

Reply:
Sure